.travis.yml

```diff
@@ -23,8 +23,7 @@ env:
 before_install:
   # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
   - rm ~/.m2/settings.xml
-  - wget -O /tmp/install.sh https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh
-  - bash /tmp/install.sh
+  - if [ ! -f ~/.jabba/jabba.sh ]; then curl -L -v --retry 5 -o jabba-install.sh https://git.io/jabba && bash jabba-install.sh; fi
```

Comment:
Hmmm, I was wrong: `https://git.io/jabba` is an alias to the `master` copy of `install.sh` so maybe not a good idea to change it. 
Still, this is the URL we're already using in playframework. 🤷🏼‍♂️

Reply:
I don't remember where I changed it, probably here, but I once send a PR to exactly not depend on the `master` version of the script.

Reply:
yep, it was here in Lagom.

https://github.com/lagom/lagom/pull/2833/files

Reply:
@ignasi35, I think we should revert to a stable version of the script, but keep the caching. 

I'll approve but I'm adding the `block-merge` label. Feel free to remove when ready.